src/lib/components/AddTransaction.svelte

```diff
@@ -49,53 +82,68 @@
     // });
 </script>
 
+<dialog id="my_modal_1" class="modal">
```

Comment:
Id "my_modal_1" does not look reliable😁

Reply:
This is some [daisyUI](https://daisyui.com/) conventions 😄 
[click](https://daisyui.com/components/modal/)